src/ceph-detect-init/Makefile.am

```diff
@@ -55,11 +55,17 @@ EXTRA_DIST += \
 
 ceph-detect-init-all: ceph-detect-init/virtualenv
 
+ifeq ($(origin CEPH_BUILD_VIRTUALENV), undefined)
+  CEPH_DETECT_INIT_VIRTUALENV = .
+else
```

Comment:
this is not valid here because it needs an `if` not an `ifeq`, or the other way around, it would need a `else ifeq`

It breaks the build too: https://jenkins.ceph.com/job/ceph-pull-requests/3461/console

    src/ceph-detect-init/Makefile.am:60: error: else without if